src/main/java/com/blamejared/crafttweaker/impl/game/MCGame.java

```diff
@@ -114,36 +113,6 @@
                 .collect(Collectors.toList());
     }
 
-    @ZenCodeType.Getter("itemTags")
-    public Collection<MCTag> getMCItemTags() {
-        return ItemTags.getCollection()
-                .getIDTagMap()
-                .keySet()
-                .stream()
-                .map(MCTag::new)
-                .collect(Collectors.toList());
-    }
-
-    @ZenCodeType.Getter("blockTags")
-    public Collection<MCTag> getMCBlockTags() {
-        return BlockTags.getCollection()
-                .getIDTagMap()
-                .keySet()
-                .stream()
-                .map(MCTag::new)
-                .collect(Collectors.toList());
-    }
-
-    @ZenCodeType.Getter("entityTags")
-    public Collection<MCTag> getMCEntityTags() {
-        return EntityTypeTags.getCollection()
-                .getIDTagMap()
-                .keySet()
-                .stream()
-                .map(MCTag::new)
-                .collect(Collectors.toList());
-    }
-
```

Comment:
Were these moved somewhere else? Not sure if I missed them in a different file

Reply:
Resolved on Discord

Reply:
IIRC the tag managers have a method to get all tags of their type.